src/mb_enhanced_cover_art_uploads/fetch.ts

```diff
@@ -207,7 +207,12 @@ export class ImageFetcher {
                 } else if (uint32view[0] === 0x46445025) {
                     resolve('application/pdf');
                 } else {
-                    reject(new Error(`${fileName} has an unsupported file type`));
+                    const actualMimeType = resp.responseHeaders.match(/content-type:\s*([^;\s]+)/i)?.[1];
+                    if (actualMimeType?.startsWith('text/')) {
+                        reject(new Error(`Expected ${fileName} to be an image, but received text. Perhaps this provider is not supported yet?`));
```

Comment:
Maybe we should not use the filename here since it is most likely not a filename in that case?
Since the default filename is `image` that often leads to messages like `Failed to grab images: Expected image to be an image, but received text. Perhaps this provider is not supported yet?`. Probably it is already enough to put the "filename" into quotes.